be/src/exec/vectorized/cross_join_node.cpp

```diff
@@ -626,9 +634,11 @@ pipeline::OpFactories CrossJoinNode::decompose_to_pipeline(pipeline::PipelineBui
     context_params.num_left_probers = left_source->degree_of_parallelism();
     context_params.num_right_sinkers = right_source->degree_of_parallelism();
     context_params.plan_node_id = _id;
-    context_params.filters = conjunct_ctxs();
     context_params.rf_hub = context->fragment_context()->runtime_filter_hub();
     context_params.rf_descs = std::move(_build_runtime_filters);
+    // The order or filters should keep same with NestLoopJoinNode::buildRuntimeFilters
+    context_params.filters = _join_conjuncts;
+    std::copy(conjunct_ctxs().begin(), conjunct_ctxs().end(), std::back_inserter(context_params.filters));
```

Comment:
why add _join_conjuncts to conjuct_ctxs, add more comment to explain this situation.

Reply:
Filter of NestloopJoin could come from `join_conjuncts` and `conjuncts`, which is corresponding to `JOIN ON` and `WHERE` clause.